### DISCUSSION TITLE
Lepton mass window check

### DISCUSSION THREAD

**Discussion by jialin-guo1** [ram1123/2l2Q_limitSettingTool#33] [Comments_HIG_23_001] [answered]
![image](https://github.com/ram1123/2l2Q_limitSettingTool/assets/68882142/781cfc91-1262-4224-8a3c-169810d59c03)

Significant check with different mass2l window. Significant = sig/sqrt(sig+bkg)
Mass window | significant
-- | --
[60,120] | 25.73
[70,105] | 25.62
[75,100] | 25.35
[80,100] | 25.12

</div></b>

**Comment by ram1123** [comment, marked as answer]
done
